app/services/sip_arrange.service.js

```diff
@@ -67,9 +72,11 @@
             child.has_children = true;
             child.children = [];
             child.children_fetched = false;
+            child.directory = true;
           } else {
             // file
             child.has_children = false;
+            child.directory = false;
```

Comment:
What does `.directory` give us that `.has_children` doesn't?  Does the GUI depend on it?